resources/views/components/select.blade.php

```diff
@@ -141,7 +141,7 @@ class="w-5 h-5
 
     <x-wireui::parts.popover :margin="(bool) $label" root-class="sm:w-full">
         <template x-if="asyncData.api || (config.searchable && options.length > 10)">
-            <div class="px-2 my-2" wire:key="search.options">
+            <div class="px-2 my-2" :wire:key="'search.options.' . $name">
```

Comment:
It doesn't will take any effect, remove the ":" in the wire:key attribute
```suggestion
            <div class="px-2 my-2" wire:key="search.options.{{ $name }}">
```